pkg/gatewayserver/config.go

```diff
@@ -57,6 +57,8 @@ type Config struct {
 	UpdateGatewayLocationDebounceTime time.Duration `name:"update-gateway-location-debounce-time" description:"Debounce time for gateway location updates from status messages"`
 	UpdateConnectionStatsDebounceTime time.Duration `name:"update-connection-stats-debounce-time" description:"Time before repeated refresh of the gateway connection stats"`
 
+	UpdateVersionInfoDelay time.Duration `name:"update-version-info-delay" description:"Maximum time to wait to update version information. A Jitter of 100% is applied for randomization"`
```

Comment:
100% ? I'd suggest 15-25%.

Reply:
The default delay is 5s. So -100 is 0 and +100 is 10s. That's gives us a 10s range. Using 25%, the range is 3.75 - 6.25. This means that this call will always wait for at least 3.75s and I don't want that.

Reply:
The average user will wait 5 seconds anyway. The fact that the lower bound is 'lower' does not change the expected value.